Tools/CISupport/built-product-archive

```diff
@@ -241,7 +241,32 @@ def archiveBuiltProduct(configuration, platform, fullPlatform, minify=False):
         # On GTK+/WPE/JSC we don't need the intermediate step of creating a thinDirectory
         # to be compressed in a ZIP file, because we can create the ZIP directly.
         # This is faster and requires less disk resources.
-        contents = ['bin', 'lib']
+        contents = ['bin',]
+
+        # Don't pack files named with following prefixes, unless they are resolved from a symbolic
+        # link. This helps reducing the zip file size in situations where the build directory
+        # contains old library files.
```

Comment:
This is a tad convoluted, but I guess it's reasonable because the next best thing to do would be to scan the headers of the ELF binaries under `bin/` to pack only libraries listed in `DT_NEEDED` entries—but in turn that would be even more complex, so let's keep the approach introduced by your patch 😉

Reply:
I agree it's convoluted... Another approach would be to avoid those old files in the first place, teaching cmake somehow to clean-them up when API or ABI is bumped...